### DISCUSSION TITLE
Accessibility Guidelines/Checklists

### DISCUSSION THREAD

**Discussion by elangager3** [twilio-labs/paste#2955] [Q&A] [answered]
Hello! 

The team is working on putting together a subsystem library in Figma for specific designs. We want to ensure we're building out these components with accessibility in mind, so were wondering if there are any general guidelines or accessibility checklists the infrastructure team could share with us?

Please let me know if I can provide any additional context, and/or if office hours makes sense to discuss further. Thanks so much!

**Comment by loreina** [comment]
Hey @elangager3, I've brought this back to the team and we'l follow up shortly. Thanks for your patience!

**Reply by elangager3** [reply]
@loreina thanks so much! 😊

**Comment by loreina** [comment, marked as answer]
Hey @elangager3! 

The [A11Y Project's checklist](https://www.a11yproject.com/checklist/) is also a great resource and place to start, though accessibility is more of an ongoing process. We also recommend looking at our [Inclusive Design Guide](https://paste.twilio.design/inclusive-design), and [Colors Foundations page](https://paste.twilio.design/foundations/colors) for how we meet WCAG 2.1. 

You're welcome to come to our [office hours](https://docs.google.com/spreadsheets/d/10vD91zdwr9hjgDOV3HSGnoC0LgvYw-thmn3X_kvW0xk/edit#gid=664218049) for any accessibility questions you have too! Hope this helps!
